rero_ils/modules/files/api.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
```

Comment:
`Copyright (C) 2019-2024 RERO`
`# Copyright (C) 2019-2023 UCLouvain` could be removed (we didn't work on the files modules)